src/ext/java/org/opentripplanner/ext/fares/model/FareLegRule.java

```diff
@@ -8,6 +8,9 @@ public record FareLegRule(
   @Nullable String networkId,
   @Nullable String fromAreaId,
   @Nullable String toAreadId,
+  @Nullable Double minDistance,
+  @Nullable Double maxDistance,
+  @Nullable Integer distanceType,
```

Comment:
Can you please map this into a proper enum?

Reply:
Just to clarify, would this be only for the distance type? It seems that all the distance related parameters are highlighted here? And  should this enum would be placed in the `ext/fares/model` as it is only used in fares computation?

Reply:
Yes, it applies only to `distanceType` and it should be placed next to `FareLegRule`.